### ISSUE TITLE
Update email example to align with documentation case study

### ISSUE THREAD

**Issue by dezzie** [watson-developer-cloud/tone-analyzer-nodejs#73]
In the documentation there is a [case study](http://www.ibm.com/smarterplanet/us/en/ibmwatson/developercloud/doc/tone-analyzer/case-study.shtml) that requires certain tone thresholds to be hit so that @kcheat can show devs how the service can be used to edit content. Currently there's misalignment with the screenshots used, but the current text isn't showing the right scores. 

- [x] change email text to make disgusting tone likely
- [x] give updated screenshots to @kcheat

**Comment by dezzie** [comment]
Here's the updated text: 
> Hi Team,

The times are difficult! Our sales have been disappointing for the past three quarters for our data analytics product suite. We have a competitive data analytics product suite in the industry. But we are not doing a good job at selling it.

We need to acknowledge and fix our sales challenges. We cannot blame the economy for our lack of execution! We are missing critical sales opportunities. Our clients are hungry for analytical tools to improve their business outcomes. In fact, it is in times such as this, our clients want to get the insights they need to turn their businesses around. We need to buckle up and execute.

In summary, we have a competitive product, and a hungry market. We have to do our job to close the deals.

Jennifer Baker

Sales Leader, North-East region

**Comment by dezzie** [comment]
Here are the updated visuals for you to dump into the docs, @kcheat:

**start**
![original](https://cloud.githubusercontent.com/assets/1301932/15262002/49f69aea-1925-11e6-983e-5e1a0d430c71.png)

**edited**
![edited](https://cloud.githubusercontent.com/assets/1301932/15262013/54c0ebe2-1925-11e6-8626-ff9bda696bc3.png)

**Comment by dezzie** [comment]
The file is changed in the demo https://github.com/watson-developer-cloud/tone-analyzer-nodejs/commit/ec188cddf0990364c2c11370615b925e7b43d33f

**Comment by kcheat** [comment]
@dezzie thanks for the updates!! for these visuals, can you please also give me the source (in .ai or.svg)? we need them for doc translation purposes - we put the text on a different layer (called text) so that the translation teams can go in and more easily translate the text and then generate the .png for each of the languages

**Comment by dezzie** [comment]
[tone-analyzer-translate-me.ai.zip](https://github.com/watson-developer-cloud/tone-analyzer-nodejs/files/264106/tone-analyzer-translate-me.ai.zip)

**Comment by kcheat** [comment]
thanks!!!

**Comment by kcheat** [comment]
updated with the new graphics and text. thanks again @dezzie!
